claims.go

```diff
@@ -151,7 +151,7 @@ func partitionIDToString(pid int32) string {
 // Claims is the wrapper around PSA claims
 // nolint: golint
 type Claims struct {
-	Profile           *eat.Profile  `cbor:"18,keyasint" json:"profile"`
+	Profile           *eat.Profile  `cbor:"265,keyasint" json:"profile"`
```

Comment:
A general comment, should we not reference (within comments) a link to a document where the code points are defined.

Reply:
we provide the reference in the README.md